src/modules/network/implementation/libp2p-service.js

```diff
@@ -76,7 +76,7 @@ class Libp2pService {
             if (!this.config.privateKey) {
                 id = await _create({ bits: 1024, keyType: 'RSA' });
                 this.config.privateKey = id.toJSON().privKey;
-                if (process.env.NODE_ENV === 'development') {
+                if (process.env.NODE_ENV === 'testnet' || process.env.NODE_ENV === 'mainnet') {
```

Comment:
Why are we not anymore saving key in development env?

Reply:
Idea was to have fresh network every time we start new lns, but i see now that we will have problem here, node will create new network id every time node restarts. Also this is the bug we never saved network id in file on testnet, that means that we always created new one after restart.